contracts/ERC20.sol

```diff
@@ -259,7 +258,7 @@ contract ERC20 is IERC20 {
 
```

Comment:
Check (require) that account has enough funds to burn

Reply:
added